Frontend/features/orders/OrderDetail.tsx

```
@@ -1,65 +1,175 @@
-  TableBody,
-  TableCell,
-  TableHead,
-  TableRow,
-  Typography,
-} from '@mui/material';
-import dayjs from '@/lib/extended-dayjs';
+  Calendar,
+  CreditCard,
+  Ellipsis,
+  FolderPen,
+  Map,
```

Biome (1.9.4)

[error] Do not shadow the global "Map" property.

Consider renaming this variable. It's easy to confuse the origin of variables when they're named after a known global.

(lint/suspicious/noShadowRestrictedNames)